lib/tasks/one_time.rake

```diff
@@ -8,4 +8,17 @@ namespace :one_time do
   task assign_ui_and_gem_signin: :environment do
     User.where(mfa_level: :disabled).where(id: WebauthnCredential.select(:user_id)).update_all(mfa_level: :ui_and_gem_signin)
   end
+
+  desc "Hash mfa recovery codes"
+  task hash_recovery_codes: :environment do
+    User.where(hashed_mfa_recovery_codes: nil).where.not(mfa_recovery_codes: nil).find_each do |user|
```

Comment:
the default for both columns is `[]`, so there will never be a case where the columns would be nil. Same goes for the `remove_plain_text_recovery_codes` rake task.
```suggestion
    User.where(hashed_mfa_recovery_codes: []).where.not(mfa_recovery_codes: []).find_each do |user|
```